sdk/eventhub/Azure.Messaging.EventHubs/tests/Snippets/Sample04_PublishingEventsLiveTests.cs

```diff
@@ -629,7 +629,9 @@ public async Task MultipleBatches()
 
             #endregion
 
+#if !SNIPPET
```

Comment:
Was this necessary? Seems the region ends right above it so this line shouldn't even be in the snippet to begin with.

Reply:
It was necessary because the variables it was using were only declared for non-snippet scenarios.   The bigger question is why were we doing test assertions and tracking in the sample snippet... since I can't answer that and we have tests that cover this scenario, I removed the non-snippet logic.